dist/style.css

```diff
@@ -427,6 +427,84 @@
   color: #fff;
 }
 
+#contact {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+#contact .contact-text {
+  display: flex;
+  width: 327px;
+  height: 216px;
+  margin: 5rem 1rem 3rem 0;
+}
+#contact .contact-text p {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.form {
+  margin-top: 10rem;
+}
```

Comment:
- [ ] Great job on implementing the feedback from the previous reviewer; please make sure your design sticks to your chosen Figma template. Kindly fix the alignment  "margin-top: 10rem" too much. So instead of rem, use px for the design to look like your chosen template.

|Figma design| Your design |
|------------------|-----------------|
| ![Screenshot from 2022-11-01 14-27-39](https://user-images.githubusercontent.com/65129228/199230876-57cbbe39-0999-448e-8bd1-ca477af9ef0b.png) |  ![Screenshot from 2022-11-01 14-28-54](https://user-images.githubusercontent.com/65129228/199230979-5190afd8-bd2c-4191-9bfd-206d1c207890.png) |